opencti-platform/opencti-graphql/config/schema/opencti.graphql

```diff
@@ -5740,6 +5753,259 @@ input CityAddInput {
 }
 
 ################ Countries
+enum Countries {
```

Comment:
all the logic of Countries enum should be removed

Reply:
Resolved in 07dffea